llvm/test/CodeGen/PowerPC/huge-frame-size.ll

```diff
@@ -18,7 +18,7 @@ define void @foo(i8 %x) {
 ; CHECK-LE-NEXT:    oris 0, 0, 65535
 ; CHECK-LE-NEXT:    ori 0, 0, 65504
 ; CHECK-LE-NEXT:    stdux 1, 1, 0
-; CHECK-LE-NEXT:    .cfi_def_cfa_offset 32
+; CHECK-LE-NEXT:    .cfi_def_cfa_offset 4294967328
```

Comment:
This looks correct now on PPC. stack size is 4294967296 + linkagesize(32) = 4294967328 (0x100000020). Without the change, it is truncated to 0x20.